transpa.public.api.example.client/README.md

```diff
@@ -0,0 +1,6 @@
+# Salary export example
+This example shows how you can export salary information via TransPA Public API into another system. This example is an actual integration that is being used. It integrates between the TransPA Public API and another Visma API.
```

Comment:
Use "Visma TransPA Public API" instead of "TransPA Public API" to emphasize that it is part of Visma?